params-csv-composite-merge-foreign.ini

```diff
@@ -1,4 +1,4 @@
 ldbc.snb.datagen.generator.scaleFactor:snb.interactive.1
 
-ldbc.snb.datagen.mode:interactive
-ldbc.snb.datagen.serializer.format:CsvCompositeMergeForeign
\ No newline at end of file
+generator.mode:interactive
+hadoop.serializer.format:CsvCompositeMergeForeign
```

Comment:
just nitpicking, but could we add the trailing new lines to these files?
editorconfig will help you with that, just need to place a config file
https://editorconfig.org/
and a plugin for your IDE